### PR TITLE
Release notes for APID 2.4.3

### PR DESCRIPTION
**Pull request by rddheath**

### CODE REVIEW

**Comment by rddheath** [comment]
1. I'll put them in the RN for the mocking service. Putting them here was a mistake.
2. Those two bugs are dev bugs and are labeled as such. I don't document dev bugs, only bugs that were found in production.

File: modules/ROOT/pages/design-center/design-center-release-notes-api_specs.adoc [hunk 1]
```diff
@@ -18,13 +18,9 @@ These release notes cover the following versions of these features:
...
-This release fixes four issues.
+This release fixes two issues.
 
 === Fixed Issues
-// SE-10773
-* For an API specification written in RAML, the mocking service returned content that was in YAML, although the returned content should have been in XML.
-// SE-10865
-* The mocking service returned a misleading error message when `baseURI` was invalid.
 // SE-10793
 * The behavior of the *Import API Example* dialog was faulty, making a selection difficult. You can open this dialog when you first create an API-specification project by clicking the word "example" in the sentence "Start project from an existing example," which appears in the left pane of the editor
```
Comment (dhirsch1) [inline modules/ROOT/pages/design-center/design-center-release-notes-api_specs.adoc]: I don't think this section is new, but it is missing a period after the word editor.

Comment (rddheath) [reply modules/ROOT/pages/design-center/design-center-release-notes-api_specs.adoc]: Well, it's too late to correct in this PR. As you can see from notes above, I merged the changes and deleted the branch. If you feel strongly enough that this period needs to be added, please open a DOCS ticket for me to do so.

**Comment by dhirsch1** [comment]
I do not feel strongly. I will approve the request.

**Pull request merged**